src/settings-ui/Settings.UI/SettingsXAML/OOBE/Views/OobeOverview.xaml

```diff
@@ -9,20 +9,16 @@
     mc:Ignorable="d">
 
     <custom:OOBEPageControl x:Uid="Oobe_Overview" HeroImage="ms-appx:///Assets/Settings/Modules/OOBE/OOBEPTHero.png">
-
         <custom:OOBEPageControl.PageContent>
-            <StackPanel Orientation="Vertical">
+            <StackPanel Orientation="Vertical" Spacing="12">
                 <TextBlock x:Uid="Oobe_HowToUse" Style="{ThemeResource OobeSubtitleStyle}" />
```

Comment:
```suggestion
```

We could get rid of this right? The link is now next to the Settings button so the title doesn't really have a purpose anymore?

![image](https://github.com/microsoft/PowerToys/assets/9866362/93db0886-0207-4b14-a9d1-db326c2ebc8e)